frontend/src/components/ingress/ClassDetails.tsx

```diff
@@ -0,0 +1,25 @@
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+import IngressClass from '../../lib/k8s/ingressClass';
+import { DetailsGrid } from '../common/Resource';
+
+export default function IngressClassDetails() {
+  const { name } = useParams<{ name: string }>();
+  const { t } = useTranslation('glossary');
+
+  return (
+    <DetailsGrid
+      resourceType={IngressClass}
+      name={name}
+      withEvents
+      extraInfo={item =>
+        item && [
+          {
+            name: t('translation|Default'),
+            value: item.isDefault ? 'Default' : '',
```

Comment:
I see that the Default column is translated, but the value in the columns is not. Maybe in this case we just add a translatable "Yes" and "No" according to the result.